### ISSUE TITLE
add package: Add verbosity option

### ISSUE THREAD

**Issue by evil-shrike** [dotnet/cli#8092]
`dotnet restore` [supports](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x) verbosity option:
```
dotnet restore --verbosity q
```
It'd nice to have the same option for `add package`:
```
dotnet add package MyPackage --versobity q
```
Currently it reports an error:
"Please specify one package reference to add"

Currently output from installing package is very noisy. Especially if compare it with npm/yarn.

**Comment by livarcocc** [comment]
@rohit21agrawal does add package support verbosity on the NuGet side?

**Comment by livarcocc** [comment]
@evil-shrike what is the point of having more verbosity for add package? what are you trying to discover that verbosity for this particular command would help you with?

**Comment by evil-shrike** [comment]
@livarcocc the point is to have less verbosity ) . It's too noisy. I think by default it's enought to output exact version installed and a chosen source.

**Comment by livarcocc** [comment]
Got it. @rohit21agrawal this is implement on NuGet side. Should I move the issue there?

**Comment by livarcocc** [comment]
Hum... looking at the code, it is possible that the verbosity is caused by the CLI when generating the DGFile. Will investigate more.

**Comment by livarcocc** [comment]
Blocked by https://github.com/NuGet/Home/issues/6654.
